data/json/items/gun/500.json

```diff
@@ -60,15 +60,7 @@
     "durability": 8,
     "blackpowder_tolerance": 56,
     "clip_size": 5,
-    "valid_mod_locations": [
-      [ "accessories", 2 ],
-      [ "grip", 1 ],
-      [ "mechanism", 4 ],
-      [ "rail", 1 ],
-      [ "sights", 1 ],
-      [ "stock mount", 1 ],
-      [ "underbarrel mount", 1 ]
-    ],
+	"build_in_mods": [ "muzzle_brake" ],
```

Comment:
There should still be a muzzle slot on this, so people don't have to scroll all the way down the item description to see that it's there.